misk/src/main/kotlin/misk/web/interceptors/RealLogRateLimiter.kt

```diff
@@ -0,0 +1,43 @@
+package misk.web.interceptors
+
+import misk.ratelimit.RateLimiter
+import misk.web.interceptors.LogRateLimiter.LogBucketId
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Holds rate limiters for logging success and error responses. There is a rate limiter for every
+ * action and service that calls it, for both success and error. The rate limiters are created
+ * according to the value set in [LogRequestResponse] annotation.
+ */
+@Singleton
+class RealLogRateLimiter @Inject constructor(
+  private val rateLimiterFactory: RateLimiter.Factory
+): LogRateLimiter {
+  private val actionToRateLimits = HashMap<String, RateLimitValues>()
+  private val rateLimiters = ConcurrentHashMap<LogBucketId, RateLimiter>()
+
+  override fun addRateLimits(actionClass: String, rateLimiting: Long, errorRateLimiting: Long) {
```

Comment:
I don't think you need this. I think the signature to `tryAcquire` could be 

```kotlin
tryAcquire(bucketId: LogBucketId, ratePerSecond: Long): Boolean
```

and you can get rid of `actionToRateLimits` and `RateLimitValues` entirely. You are doing a lot of bookkeeping for information that can just be passed in